packages/odyssey-react-mui/src/FieldComponentProps.ts

```diff
@@ -64,3 +85,15 @@ export type FieldComponentRenderProps = {
   id: string;
   labelElementId: string;
 };
+
+export type ConditionalLabelProps =
+  | {
+      hasVisibleLabel: false;
+      hint?: never;
+      placeholder: string;
+    }
```

Comment:
Are we expecting they at least pass in `aria-labelledby` or `aria-label` if they have no visible label?

Reply:
the actual `label` prop is still required. It's just visually hidden

Reply:
Ah, my bad. I see. So we're just saying if the label is visible, not if it doesn't exist. My bad.

We should also require `aria-labelledby` if no label is passed in as an alternative. We can keep visually hiding the label, but it'd be good to say our intent is that they have to have a visual label somewhere else. That's what I mean.

Reply:
`aria-labelledby` will be controlled by us. It'll still be labelled by the non-visible label